.github/ISSUE_TEMPLATE/7-t-minus-0-third-batch.yaml

```diff
@@ -0,0 +1,23 @@
+name: T-minus zero third batch checklist
+description: xxx.
+title: "[Release vx.xx]: T-minus zero third batch checklist"
+assignees:
+  - knative-release-leads
+body:
+- id: release-4
+  type: checkboxes
+  attributes:
+    label: Please change the title to include the current release.
+    description: |
+      - This release checklist tracks the third batch of core repos to be released.
+      - Please only mark a repo as done after it has been verified the release has been published. Release notes can be done later.
+      - View [documentation](https://github.com/knative/release/blob/main/PHASES.md#t-minus-zero-third-batch).
+    options:
+      - label: All items on the "T-minus zero second batch checklistt" have been checked off
+        required: true
+      - label: knative.dev/client
```

Comment:
I sorted mindlessly grouped them following the original grouping. See screenshot below.

But now that you pointed this out... you are correct. This third batch requires both `eventing` and `serving` to be done, and the second batch only requires `eventing`.  But since both `eventing` and serving` are on batch 1, then this can roll into batch 2.

Or I can organize it some other way. Thinking/suggestions accepted.

<img width="941" alt="CleanShot 2022-06-03 at 14 39 51@2x" src="https://user-images.githubusercontent.com/16508/171957522-f2d2f74f-3dc7-4840-af6d-c40aa2fc8ff5.png">

Reply:
I "fixed" this by mostly reverting to the way the original way things were laid out.